### PR TITLE
Poisoning scenario

### PR DESCRIPTION
**Pull request by ng390**
Partial split of PR #350.

### CODE REVIEW

**Comment by ng390** [comment]
Now tested locally and with GPU.

File: .github/workflows/ci_test.yml [hunk 1]
```diff
@@ -11,10 +11,6 @@ on:
     branches:
       - master
       - r*
-
-env:
-  DOCKER_BUILDKIT: 1
-
```
Comment (seanpmorgan) [inline .github/workflows/ci_test.yml:17]: Hmmm don't think this file should have been modified?

File: armory/baseline_models/keras/keras_gtsrb_micronnet.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
+"""
```
Comment (seanpmorgan) [inline armory/baseline_models/keras/keras_gtsrb_micronnet.py:1]: Rename the file to `gtsrb_micronnet` keras_ is redundant since the directory

File: armory/baseline_models/keras/keras_gtsrb_micronnet.py [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+        loss=tf.keras.losses.sparse_categorical_crossentropy,
+        metrics=["accuracy"],
+    )
+
+    return model
+
+
+def get_art_model(model_kwargs, wrapper_kwargs, weights=None):
+    model = make_model(**model_kwargs)
+    wrapped_model = KerasClassifier(model, clip_values=(0.0, 255.0), **wrapper_kwargs)
```
Comment (seanpmorgan) [inline armory/baseline_models/keras/keras_gtsrb_micronnet.py:67]: Curious as to why we don't normalize input? Can be changed later on though

File: armory/baseline_models/pytorch/cifar.py [hunk 4]
```diff
@@ -46,10 +46,6 @@ def make_cifar_model(**kwargs):
 
 def get_art_model(model_kwargs, wrapper_kwargs, weights_file=None):
     model = make_cifar_model(**model_kwargs)
-
-    if torch.cuda.is_available():
-        model.cuda()
```
Comment (seanpmorgan) [inline armory/baseline_models/pytorch/cifar.py:51]: Reason to remove these? We do need to change this method to match the way it's device placement is done, but probably not this PR

File: tests/configs/pytorch/fgm_attack_test.json [hunk 5]
```diff
@@ -1,36 +0,0 @@
-{
```
Comment (seanpmorgan) [inline tests/configs/pytorch/fgm_attack_test.json:1]: Why remove?
